WcaOnRails/app/views/translators_mailer/notify_translators_of_changes.html.erb

```diff
@@ -0,0 +1,10 @@
+<p>Dear translators,</p>
+
+<p>
+  Changes have been introduced recently and some translation work is needed.
+  Please update your translation at your convenience.
```

Comment:
Might be nice to include a link to the [translations status page](https://www.worldcubeassociation.org/translations/status) here.